docs/dev-guide/quickstart.md

```diff
@@ -0,0 +1,130 @@
+# Quickstart
```

Comment:
Nit: should we label this "Development quickstart", or something to make it clear that this is development only? Someone searching for "quickstart" might find this and get scared off by this being all technical. Who knows if that'd happen in practice though 🤷

Reply:
I think that this would tie in nicely with the content reorganization of top-level headers, which would make it very clear what section of the docs you are viewing. Changed in 534c154ac9555dfbd627d9c6125483e09d0cc8de.